README.adoc

```diff
@@ -235,8 +235,8 @@ $ curl "http://$(oc get route quickstart-loadbalancer --template='{{ .spec.host
 
 ## System Requirements
 
-* https://github.com/golang/go[go] with `$GOPATH` set to `$HOME/go`
-* Docker
+* Go 1.21
```

Comment:
This of course have potential to become obsolete. Is it possible to somehow write it in dynamic way, so reader can find out himself. But as I understand rule is complex. But something like

```
Go version brought in by Operator SDK to support oldest Kubernetes version. E.g. for Kubernetes 1.25 it is Go 1.21.
```

Reply:
@mchoma We will update it accordingly when we upgrade the Operator SDK, eventually, once we get rid of the issue compiling the controller tools, this will become the latest Go version.

 "Go version brought in by Operator SDK to support oldest Kubernetes version" is even more confusing to me, since it is difficult for the reader to find which version is that one.